server/supernova/server/main.cpp

```diff
@@ -155,46 +157,45 @@ void start_audio_backend(server_arguments const& args) {
     int output_channels = args.output_channels;
 
     std::string input_device, output_device;
-    if (args.hw_name.empty()) {
-        boost::tie(input_device, output_device) = instance->default_device_names();
-    } else if (args.hw_name.size() == 1) {
-        input_device = output_device = args.hw_name[0];
-    } else {
-        input_device = args.hw_name[0];
-        output_device = args.hw_name[1];
+    if (!args.hw_name.empty()) {
```

Comment:
maybe good to comment here or below that leaving `input_device`/`output_device` being empty means that default devices are chosen